cmd/iriscli/main.go

```diff
@@ -128,7 +131,40 @@ func main() {
 		upgradeCmd,
 	)
 
-	//Add keys and version commands
+	recordCmd := &cobra.Command{
+		Use:   "record",
+		Short: "Record subcommands",
+	}
+
+	recordCmd.AddCommand(
+		recordcmd.GetCmdQureyHash(cdc),
+	)
+
+	recordCmd.AddCommand(
+		client.GetCommands(
+			recordcmd.GetCmdDownload(cdc),
+		)...)
+
```

Comment:
`recordCmd.AddCommand(
		client.GetCommands(
                         recordcmd.GetCmdQureyHash(cdc),
			recordcmd.GetCmdDownload(cdc),
		)...)`

Reply:
OK, got it.